tools/manifest/sourcefile.py

```diff
@@ -30,6 +30,8 @@
 
 wd_pattern = "*.py"
 js_meta_re = re.compile(br"//\s*META:\s*(\w*)=(.*)$")
+# Matching <link rel="help" href="https://www.w3.org/TR/accelerometer/">.
+js_link_rel_re = re.compile(br"<link\s*rel=\"?help\"?\s.*href=\"(.*)\".*>$")
```

Comment:
This pattern is HTML, but I think the other metdata in HTML is extracted by parsing it, like this bit for variants:
https://github.com/web-platform-tests/wpt/blob/bf6e47b699ba0d78c26bb3a00d611569d0e02a0a/tools/manifest/sourcefile.py#L685

Reply:
Addressed in https://github.com/web-platform-tests/wpt/pull/40524